.gitignore

```diff
@@ -18,6 +18,9 @@ bld/
 # Visual Studio 2015/2017 cache/options directory
 .vs/
 
+#package
+packages/
```

Comment:
[Pp]ackages/
But, have you test this again? are you sure if you delete this, your project still working?

Reply:
em cũng không biết! nhưng em nghĩ nó như nodeModule trong react , chỉ cần gõ lệnh hay auto gì đó nó sẽ import vào lại! Em thấy file References có tất cả thư viện trong đó , chắc nó sẽ lấy trong này ra